src/sonic-host-services/scripts/hostcfgd

```diff
@@ -537,23 +589,8 @@ class HostConfigDaemon:
         self.iptables.iptables_handler(key, data, add)
         self.ntpcfg.handle_ntp_source_intf_chg(key)
 
-    def feature_state_handler(self, key, data):
-        feature_name = key
-        feature_table = self.config_db.get_table('FEATURE')
-        if feature_name not in feature_table:
-            syslog.syslog(syslog.LOG_WARNING, "Feature '{}' not in FEATURE table".format(feature_name))
-            return
-
-        state = feature_table[feature_name]['state']
-        if not state:
-            syslog.syslog(syslog.LOG_WARNING, "Enable state of feature '{}' is None".format(feature_name))
-            return
-
-        self.cached_feature_states.setdefault(feature_name, 'disabled')
-
-        # Enable/disable the container service if the feature state was changed from its previous state.
-        if self.cached_feature_states[feature_name] != state:
-            self.update_feature_state(feature_name, state, feature_table)
+    def feature_handler(self, key, data):
+        self.feature.handle(key, data)
```

Comment:
Is this function necessary? Is it possible to rename the instantiated object `feature_hander` and pass `self.feature_handler.handle(key, data)` as the callback on line 621? Can you see if that works?

Reply:
Not necessary, thanks for suggestion.